### DISCUSSION TITLE
Having Trouble Installing

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [d0k3/GodMode9#843] [Ideas]
I'm trying to hack my old 2DS. I can't get GM9 to open, it is just booting right to the homescreen. I believe I've followed the instructions properly. 

Here's a [link](https://paste.samgreenwood.ca/?5f839c2d221c2837#A22XJCbTs4AiQ8QAxgb3nEuDdCpkKk9sMWPskuhjAdtu) to an export of the file tree of the SD card

**Comment by ihaveamac** [comment]
Try holding SELECT at boot to get to Luma3DS configuration. If that doesn't appear, you probably did not set up custom firmware fully (you need this to run GodMode9). A guide for this at at https://3ds.hacks.guide

**Reply by SamGreenwood1** [reply]
I followed that guide exactly.

**Reply by urherenow** [reply]
No you did not. I don't see a single .3dsx file, let alone boot.3dsx.

**Reply by SamGreenwood1** [reply]
I realized that I followed a [different guide](https://www.cfwaifu.com/3ds-dump-cartridges/), though I did look at part of one you shared just didn't realize that I wasn't at the beginning.
